core/pom.xml

```diff
@@ -614,6 +614,12 @@ THE SOFTWARE.
       <artifactId>jzlib</artifactId>
       <version>1.1.3-kohsuke-1</version>
     </dependency>
+    <!-- Should be removed after splitting maven-builder-plugin see JENKINS-26462 -->
+    <dependency>
```

Comment:
:bug: for lack of conversion of `ProjectWithMaven`. This was after all the predicted main work for this PR. :-)